src/BuildValues.props

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Seems like notepad added BOM at the begining of the file. It can be removed using [HexEdit](http://hexedit.com/) on Windows. While at it, this file is also missing end of file linefeed. :)